feature/login.feature

```diff
@@ -0,0 +1,64 @@
+Feature: Login no aplicativo
+    As a usuario do aplicativo
+    I want to entrar no aplicativo com meu e-mail e senha cadastrado
+    So that eu tenha acesso às funcionalidades do sistema que são acessíveis somente depois do login
+
+Scenario: Login correto
+    Given o cliente está na página de "login"
+        And existe um usuario cadastrado com o email "USER_EMAIL" e senha "USER_SENHA"
+        And estou na página inicial de "login"
+    When eu preencho os campos email e senha corretamente
+    Then o cliente é encaminhado para a tela inicial do aplicativo
+
+Scenario: Login admin
+    Given o admin não está logado com nenhum usuário na aplicação
+        And existe um admin cadastrado com o email " ADMIN_EMAIL" e senha "ADMIN_SENHA"
+        And estou na página inicial de "login"
+    When eu preencho os campos email e senha corretamente
+    Then o admin é encaminhado para a pagina do administrador
+
+Scenario: Login com a senha errada
```

Comment:
The scenario have sensible data, follow the pattern for environment variables suggested by @acrucha.

Reply:
Same commet for the following ones.